src/licensedcode/models.py

```diff
@@ -2101,6 +2126,10 @@ def from_file(cls, rule_file, is_builtin=True):
         rule.load_data(rule_file=rule_file)
         return rule
 
+    @property
+    def pysafe_expression(self):
+        return python_safe_name(self.license_expression)
```

Comment:
Add a docstring to explain why you need this?

Reply:
added, thanks